tls/extensions/s2n_server_supported_versions.c

```diff
@@ -43,7 +43,7 @@ static int s2n_server_supported_versions_recv(struct s2n_connection *conn, struc
 
 const s2n_extension_type s2n_server_supported_versions_extension = {
     .iana_value = TLS_EXTENSION_SUPPORTED_VERSIONS,
-    .is_response = true,
+    .is_response = false,
```

Comment:
Why did you need to change this to false? Would we ever receive the s2n_server_supported_versions_extension extension without the s2n_client_supported_versions_extension?

If it's just because a test failed, use https://github.com/awslabs/s2n/blob/master/tests/testlib/s2n_testlib.h#L59 to pretend all request extensions were sent/received.